tests/base.py

```diff
@@ -14,9 +12,10 @@ class BaseTestClass:
 
     @classmethod
     def setup_class(cls):
-        if not os.path.exists(AUTH_TOKEN_DIR):
+        token_file = os.path.join(AUTH_TOKEN_DIR, AUTH_TOKEN_FILE_NAME)
+        if not os.path.exists(token_file):
             os.makedirs(AUTH_TOKEN_DIR)
-            with open(osp.join(AUTH_TOKEN_DIR, AUTH_TOKEN_FILE_NAME), 'w') as fw:
+            with open(token_file, 'w') as fw:
```

Comment:
Is the token saved as plaintext? Should it be encrypted in any way?

Reply:
yes, it is! But yes I think we should encrypt it. @isht3 Can you please add that change in a new PR.

Reply:
We don't need to encrypt the token since it's already encrypted by the django-rest-auth package.